jenkins.io.tf

```diff
@@ -0,0 +1,20 @@
+resource "azurerm_resource_group" "jenkins_io" {
+  name     = "jenkinsio"
+  location = var.location
+}
+
+resource "azurerm_storage_account" "jenkins_io" {
+  name                     = "jenkinsio"
+  resource_group_name      = azurerm_resource_group.jenkins_io.name
+  location                 = azurerm_resource_group.jenkins_io.location
+  account_tier             = "Premium"
```

Comment:
There are missing parameters (some are mandatory otherwise the creation won't succeed despite the terraform plan): see https://github.com/jenkins-infra/azure/blob/5fc5c80a3a97901ad5c99041b375419c0db9dd45/get.jenkins.io.tf#L13-L38

Reply:
Thanks, adding them